modules/apps/content-dashboard/content-dashboard-web/test/js/components/AuditBarChart.js

```diff
@@ -177,6 +177,47 @@ const mockTwoVocabulariesWithCategoriesInTheFirstVocabulary = [
 	},
 ];
 
+const mockTwoVocabulariesWithNoneCategory = [
+	{
+		categories: [
+			{
+				key: 'education',
+				name: 'Education',
+				value: 478,
+				vocabularyName: 'Stage',
+			},
+			{
+				key: 'selection',
+				name: 'Selection',
+				value: 1055,
+				vocabularyName: 'Stage',
+			},
+			{
+				key: 'none',
```

Comment:
Have you tested not using the last position in the mocked data?

Reply:
Yes, the test doesn't assert the sorting, it only checks if the none name is displayed in the UI: "No Stage Specified".